packages/api/lambdas/bootstrap.js

```diff
@@ -95,11 +95,7 @@ async function bootstrapElasticSearch(host, index = 'cumulus', alias = defaultIn
       index: index,
       name: alias
     });
-
-    log.info(`index ${index} created with alias ${alias} and mappings added.`);
```

Comment:
So I know these can be annoying, but they actually are kind of helpful for debugging issues with the indices and deployments. I would prefer these stay in.

Reply:
Just this one, or all of them that I removed?  Some were getting hit many many times per test on the dashboard.  But none are as bad as our new deprecated functions....

Reply:
Just in this file. These two.

Reply:
4b380209a